File: kotlin/classTest3.kt
class Person{
    var name:String
    var age:Int
    
    constructor(name:String, age:Int){
        this.name=name
        this.age=age
    }

    fun say(){
    	println("$name $age")
    }
}

fun test(){
    var a = Person("Stanley", 30)
    a.say()
}

File: swift/classTest3.swift
class Person<R, S>{
    let name:R
    let age:S
    
    init(name:R, age:S){
        self.name = name
        self.age = age
    }
    
    func say(){
        print(self.name, self.age)
    }
}

func test(){
    let a = Person(name:"Stanley", age:20)
    a.say()
}

test()